content/docs/rfcs/template/README.md

```diff
@@ -26,22 +27,26 @@ Currently identified tags comprise
 * `waku/application` for applications built on top of Waku protocol (e.g. eth-dm, toy-chat),
 
 
-# Abstract
+## Abstract
```

Comment:
Context:

> Reasons being,
> 
> - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page not the best practice
> - https://github.com/DavidAnson/markdownlint/blob/v0.31.1/doc/md025.md
> - h1 is already the title from the frontmatter
> - formatting on the new Status website
>
> – https://discord.com/channels/864066763682218004/1179553924617871422/1181242144556593275

> Then let's have the discussion 🙂. Shouldn't be that challenging.
>
> – https://discord.com/channels/864066763682218004/1179553924617871422/1181243815735078992

Reply:
some of the files not following it atm

- content/specs/rfc/content/docs/rfcs/10/README.md
- content/specs/rfc/content/docs/rfcs/8/README.md
- content/specs/rfc/content/docs/rfcs/14/README.md
- content/specs/rfc/content/docs/rfcs/2/README.md
- content/specs/rfc/content/docs/rfcs/26/README.md
- content/specs/rfc/content/docs/rfcs/3/README.md
- content/specs/rfc/content/docs/rfcs/35/README.md
- content/specs/rfc/content/docs/rfcs/38/README.md
- content/specs/rfc/content/docs/rfcs/4/README.md
- content/specs/rfc/content/docs/rfcs/5/README.md
- content/specs/rfc/content/docs/rfcs/6/README.md
- content/specs/rfc/content/docs/rfcs/7/README.md
- content/specs/rfc/content/docs/rfcs/18/README.md
- content/specs/rfc/content/docs/rfcs/43/README.md
- content/specs/rfc/content/docs/rfcs/12/README.md